cross/android/arm64/tryrun.cmake

```diff
@@ -0,0 +1,128 @@
+# Need to verify this
+SET( HAVE_COMPATIBLE_EXP_EXITCODE 
```

Comment:
I assume these constants were tested on Android and not just copied from their Linux versions, right?